src/annotator/test/guest-test.js

```diff
@@ -48,7 +48,8 @@ describe('Guest', () => {
 
   const createGuest = (config = {}) => {
     const element = document.createElement('div');
-    return new Guest(element, { ...guestConfig, ...config });
+    const eventBus = new EventBus();
```

Comment:
Generally it is preferable for tests to avoid accessing the private API (eg. `_emitter`) for the classes they are testing. Here you could avoid that by creating the `EventBus` in the `beforeEach`, saving it as an `eventBus` variable in the scope of the top-level describe block and then calling `eventBus.emit(...)` in individual tests. That wouldn't really have any immediate practical benefits, although it might discourage future tests from using private APIs when they shouldn't be.

Reply:
I followed your suggestion of no accessing private properties.

Reply:
Furthermore, it would be confusing to expose the internal API of the private TinyEmitter class. Generally, developers should use the API of `Emitter`.